private/manageFishData.py

```diff
@@ -160,7 +165,17 @@ def initialize_data(args):
                'region_id': territory.region_place_name.key,
                'weather_rates': _collect_weather_rates(territory.weather_rate)}))
         for territory in TERRITORIES])
-    
+
+    GATHERING_SUB_CATEGORIES = dict([
```

Comment:
I think we could obtain this similarly to how the aquarium data is gathered.
Keep in mind, the saintcoinach library handles caching and such.  So, unless the table is going to be forwarded to the JavaScript, it doesn't need to be "preloaded", and such.

Reply:
I tried to do it in another way but than i could only get "Othardian fishing folklore" instead of the needed item "Tome of Ichthyological Folklore - Othard" and i am not that familiar with some of the magic you are doing while loading^^'

Reply:
Inside the `convert_fish_to_json` function:
```python
fish_params = first(XIV.game_data.get_sheet('FishParameters'), lambda r: r.get_raw('Item') == key)
if fish_params is not None:
    folklore = fish_params.get_raw('GatheringSubCategory')
    if folklore == 0:
        folklore = None
```
Store the ID in the `folklore` field so it can be looked up in another table on the site (for localization).

Keep the GATHERING_SUB_CATEGORY, but define it as:
```python
    GATHERING_SUB_CATEGORIES = dict([
        (x.key,
         dict([*_make_localized_field('book', x, 'FolkloreBook'),
               *_make_localized_field('name', x['Item'], 'Name')]))
        for x in XIV.game_data.get_sheet('GatheringSubCategory')])
```

This will generate a dictionary containing the keys: `book_*` and `name_*`.  The site can use the ID to lookup the correct entry, and 0 to know it's nothing.

Reply:
Applied the provided patch below, so this is completed.